templates/includes/CartActions.ss

```diff
@@ -0,0 +1,16 @@
+<% if $isAllowedInCart %>
+    <div class="dms-cart-actions">
+        <p>
+            <a class="<% if not $isInCart %>hidden<% end_if %>"
+               href="$getActionLink('remove')"><%t DMSCart.REMOVE_FROM_CART "Remove from cart" %> </a>
+            <a class="<% if $isInCart %>hidden<% end_if %>"
+               href="$getActionLink('add')"><%t DMSCart.ADD_TO_CART "Request a printed copy" %> </a>
+            <% if $isInCart %>
+                &bull;
```

Comment:
Can you add this with CSS instead? Otherwise people will have to override this template to be able to style it